examples/rollbar-spring-webmvc/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
I think we don't need this, as we should use the gradle wrapper script from the root project level.
So this file and the gradlew.bat are not needed nor the `examples/rollbar-spring-webmvc/settings.gradle`

Reply:
done